cpp/src/utilities/spmv_1D.cuh

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <rmm/thrust_rmm_allocator.h>
+#include <raft/handle.hpp>
+#include "utilities/cusparse_helper.h"

```

Comment:
Better use RAFT cuSparse wrapper.

Reply:
Yup, that's what the code comment on the next line explains.

```
// FIXME : use RAFT cusparse wrapper
// The wrapper is about to be upgraded in RAFT
// better wait for it to be stable before building on top of it
// #include <raft/sparse/cusparse_wrappers.h>
```

Reply:
2d430fc

Reply:
Just waiting on RAFT cuSparse wrapper to be stable, there are likely two other PR coming before we close #485.